app/src/styles.sass

```diff
@@ -131,15 +132,16 @@ input
 .information-container
   display: grid
   grid-template-columns: 1fr minmax(min-content, 5%)
+  @media screen and (min-width: $breakpoint-laptop)
+    grid-template-columns: 1fr minmax(min-content, 31%)
```

Comment:
It should be in the other direction: by default the max-width of the second column is 31%, and on narrow screen it is shrunken to 5%.